pkg/validation/cookie.go

```diff
@@ -7,18 +7,19 @@ import (
 
 	"github.com/oauth2-proxy/oauth2-proxy/v7/pkg/apis/options"
 	"github.com/oauth2-proxy/oauth2-proxy/v7/pkg/encryption"
+	"github.com/oauth2-proxy/oauth2-proxy/v7/pkg/util"
 )
 
-func validateCookie(o options.Cookie) []string {
+func validateCookie(o *options.Cookie) []string {
 	msgs := validateCookieSecret(o.Secret)
 
-	if o.Refresh >= o.Expire {
-		msgs = append(msgs, fmt.Sprintf(
-			"cookie_refresh (%q) must be less than cookie_expire (%q)",
-			o.Refresh.String(),
-			o.Expire.String()))
+	rv := util.RefreshValue{}
```

Comment:
I feel like this needs a `NewRefreshValue(s string)` instead of the existing `Set` method that wants a default struct.

Reply:
Sure.